src/Nethermind/Nethermind.Network.Stats/Model/Node.cs

```diff
@@ -227,6 +227,10 @@ public static NodeClientType RecognizeClientType(string clientId)
             {
                 return NodeClientType.EthereumJS;
             }
+            else if (clientId.Contains(nameof(NodeClientType.Ethrex), StringComparison.OrdinalIgnoreCase))
```

Comment:
Do switch on the first letter to reduce the number of comparisions? (Maybe Regex would be better?)

Reply:
I do think this one can be improved with single compiled `Regex` pass?